frappe/desk/reportview.py

```diff
@@ -263,18 +263,33 @@ def delete_bulk(doctype, items):
 @frappe.whitelist()
 @frappe.read_only()
 def get_sidebar_stats(stats, doctype, filters=[]):
-	_user_tags = []
+	_user_tags, tag_list = [], []
 	data = frappe._dict(frappe.local.form_dict)
 	filters = json.loads(data["filters"])
+	with_child_table_filter = ""
+
+	# Show Tags irrespective of any tag filter set
+	for idx, flt in enumerate(filters):
+		if flt[0] != "Tag Link" or flt[0] != doctype:
```

Comment:
```suggestion
		if filter[0] == 'Tag Link':
			filters.pop(idx)
		elif len(filter) == 4 and filter[0] == doctype:
```
**Note:** It's better to push valid filter to a new list instead of poping from the current one